sdk/core/azure-core/inc/azure/core/url.hpp

```diff
@@ -40,8 +40,9 @@ namespace Azure { namespace Core {
 
   /**
    * @brief Represents the location where a request will be performed.
-   * It can be parsed and initialized from a string that contains all URL components (scheme, host,
-   * path, etc.). Authority is not currently supported.
+   *
+   * @details It can be parsed and initialized from a string that contains all URL components
+   * (scheme, host, path, etc.). Authority is not currently supported.
```

Comment:
> Authority is not currently supported.

This is unrelated to the PR, but really? How come? What happens if someone creates a url (or provides a string) with authority? Do we have a test for it?

Reply:
The URL is currently only for Azure Services.  
I don't expect people using it to parse any URL in the world and to use it independently from the Azure SDK.

I would expect that if any Azure Service uses/requires URLs with the authority in it, then Azure Core will be updated to support it. Right now, it's not required.